models/prices/prices_tokens.sql

```diff
@@ -581,11 +581,14 @@ VALUES
     ("eure-monerium-eur-emoney", "ethereum", "EURe", "0x3231Cb76718CDeF2155FC47b5286d82e6eDA273f", 18),
     ("fort-forta", "ethereum", "FORT", "0x41545f8b9472D758bB669ed8EaEEEcD7a9C4Ec29", 18),
     ("caps-ternoa", "ethereum", "CAPS", "0x03be5c903c727ee2c8c4e9bc0acc860cca4715e2", 18),
+    ("aura-aura-finance",  "ethereum", "AURA", "0xc0c293ce456ff0ed870add98a0828dd4d2903dbf", 18),
+    ("inv-inv", "ethereum", "INV", "0x41d5d79431a913c4ae7d69a668ecdfe5ff9dfb68", 18),    
+    ("fdt-fiat-dao-token", "ethereum", "FDT", "0xed1480d12be41d92f36f5f7bdd88212e381a3677", 18),      
+    ("silo-silo-finance", "ethereum", "Silo", "0x6f80310ca7f2c654691d1383149fa1a57d8ab1f8", 18),    
     ("strp-strips-finance", "ethereum", "STRP", "0x97872eafd79940c7b24f7bcc1eadb1457347adc9", 18),
     ("wsteth-wrapped-liquid-staked-ether-20", "ethereum", "wstETH", "0x7f39c581f595b53c5cb19bd0b3f8da6c935e2ca0", 18),
     ("reth-rocket-pool-eth", "ethereum", "rETH", "0xae78736cd615f374d3085123a210448e74fc6393", 18),
     ("temple-templedao", "ethereum", "TEMPLE", "0x470ebf5f030ed85fc1ed4c2d36b9dd02e77cf1b7", 18),
-
     ("dai-dai", "gnosis", "WXDAI", "0xe91d153e0b41518a2ce8dd3d7944fa863463a97d", 18),
```

Comment:
This line was intended to separate tokens from different networks.

Reply:
Oh my bad, will reintroduce it in the next PR